server/game/player.js

```diff
@@ -896,6 +896,19 @@ class Player extends Spectator {
         });
     }
 
+    moveCardToBottomOfDeck(card, allowSave = true) {
```

Comment:
I suspect @ystros might suggest that this work similarly to the above return to hand method and to factor out some of this common code